src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -6144,27 +6134,14 @@ public void SetProperty(string path, PSObject propertyToSet)
 
             PSObject results = new PSObject();
             PSObject fileSystemInfoShell = null;
```

Comment:
This variable now should be moved in `if (fsinfo != null)`, since it's only used in the if block.

Reply:
Fixed.